config/config.go

```diff
@@ -141,8 +141,9 @@ func (c PersistConfig) GetClusterName() string {
 }
 
 type ConnectionPool struct {
-	MaxIdleConns int `json:"maxIdleConns,omitempty"`
-	MaxOpenConns int `json:"maxOpenConns,omitempty"`
+	MaxIdleConns          int `json:"maxIdleConns,omitempty"`
+	MaxOpenConns          int `json:"maxOpenConns,omitempty"`
+	ConnMaxLifetimeMillis int `json:"connMaxLifetimeMillis,omitempty"`
```

Comment:
I think just `ConnMaxLifetime` should be fine. Just document that it's in milliseconds in `docs/workflow-controller-configmap.yaml`

Reply:
maybe use `time.Duration`?

Reply:
How about TTL? That's an alias for time.Duration that we have a json unmarshaller defined for, and this technically is a TTL.